### DISCUSSION TITLE
Is there an option to generate rss feed for products ?

### DISCUSSION THREAD

**Discussion by ufoloko** [openstore-ecommerce/OpenStore#265] [General]
I would like to have rss feed for products for integration with marketing campaigns, with rss feed i can detect for changes in products to use for new campaigns.

**Comment by DNNMonster** [comment]
I like the idea.  I think it may require a plugin at this time.  I could see two pathways.  You could hook into the AfterProductSave event or mimic the approach used by the scheduler indexing plugin which detects updated products to populate the Lucene index.  If there is something already available someone might share that info.  

I explored the code found here https://www.dnnsoftware.com/community-blog/cid/136023/dotnetnuke-tip-add-an-rss-feed-for-your-page-head-tag which works as advertised when implemented in the ProductAjaxView but I'm not sure it offers any added value and it's not exactly relevant to creating a feed.xml.

**Comment by leedavi** [comment]
My thought would be aplugin, based on the functionality of the DNN product index plugin.

https://github.com/openstore-ecommerce/OS_DnnIdx

Create a plugin with this base to output the required RRS,
